composer/datasets/c4_hparams.py

```diff
@@ -38,8 +40,9 @@ class StreamingC4Hparams(DatasetHparams):
         timeout (float): How long to wait for shard to download before raising an exception. Default: 120 sec.
     """
 
+    version: int = hp.optional('Version of streaming (1 or 2)', default=2)
```

Comment:
Can we update the following YAML's as well with an explicit version so it's explicit?
```
composer/yamls/models/bert-base.yaml:  streaming_c4:
composer/yamls/models/bert-base.yaml:      streaming_c4:
```

Reply:
Done